src/render/inventory.rs

```diff
@@ -188,10 +206,6 @@ impl InventoryWindow {
         for element in &mut self.text_elements {
             element.clear();
         }
-        self.text_elements.clear();
-        // for element in &mut self.text_box {
-        //     element.clear();
-        // }
```

Comment:
Is this still needed?

Reply:
nope that was never needed, this is just  a remaining piece of a earlier implementation of text boxes, from when i implemented the anvil inventory